pom.xml

```diff
@@ -31,6 +31,7 @@
         <module>client/trino-cli</module>
         <module>client/trino-client</module>
         <module>client/trino-jdbc</module>
+        <module>client/trino-jdbc-core</module>
```

Comment:
logically in terms of deps core would be first ... but technically order here doesnt matter